odoo/addons/base/models/ir_ui_view.py

```diff
@@ -308,6 +308,7 @@ def replacer(m):
 
     def _inverse_arch(self):
         for view in self:
+            view.arch = view._fields['arch_db'].translate(lambda x: None, view.arch)
```

Comment:
This is utterly obscure, and I don't understand why you need this.

Reply:
Yes, it is really ugly.

In test 'test_02_reset_specific_view_controller', the arch is broken to `'<p t-att-data="no_record.exist" />'`
The extra space after `exist"`, will make the arch/arch_db make it cannot be found(`line in v.arch`) in the following code for soft fixing
https://github.com/odoo/odoo/blob/7ae60a7261a6534915da20a2daec1a9a74a26ad0/addons/website/models/ir_http.py#L351-L353

Here I choose to translate it again, so the extra space can will be removed.